pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -582,49 +525,61 @@ func (r *Reconciler) createCheCluster(ctx context.Context, kcCfg *config.RHSSO,
 		},
 	}
 
-	ownerutil.EnsureOwner(cheCluster, inst)
+	ownerutil.EnsureOwner(cheCluster, r.installation)
 	if err := serverClient.Create(ctx, cheCluster); err != nil {
```

Comment:
What do you think about using a `controllerutil.CreateOrUpdate` here instead? That would mean a refactor of this block as well - https://github.com/integr8ly/integreatly-operator/blob/15014750a4a8df7abc2fb06e5963c3c81f9d0552/pkg/controller/installation/products/codeready/reconciler.go#L230-L250